linux/snap/snapcraft.yaml

```diff
@@ -51,7 +48,8 @@ apps:
       - nextcloud
     command: |
       env LD_LIBRARY_PATH=$SNAP/usr/lib/nextcloud:$LD_LIBRARY_PATH
-      desktop-launch nextcloud
+          FONTCONFIG_FILE=$SNAP/etc/fonts/fonts.conf
+          snapcraft-preload nextcloud
```

Comment:
Yeah there were some bugs there for a while. Worth checking if they've been fixed?